site/pages/api/get-all-disruptions/[organisationId].api.ts

```diff
@@ -24,179 +17,41 @@ export interface GetDisruptionsApiRequest extends NextApiRequest {
     };
 }
 
-export const getDisruptionStatus = (disruption: Disruption): Progress => {
-    if (disruption.publishStatus === PublishStatus.draft) {
-        return Progress.draft;
-    }
-
-    if (disruption.publishStatus === PublishStatus.rejected) {
-        return Progress.rejected;
-    }
-
-    if (disruption.publishStatus === PublishStatus.pendingApproval) {
-        return Progress.draftPendingApproval;
-    }
-
-    if (
-        disruption.publishStatus === PublishStatus.editPendingApproval ||
-        disruption.publishStatus === PublishStatus.pendingAndEditing
-    ) {
-        return Progress.editPendingApproval;
-    }
-
-    if (!disruption.validity && !disruption.template) {
-        return Progress.closed;
-    }
-
-    const today = getDate();
-    const disruptionEndDate = getSortedDisruptionFinalEndDate(disruption);
-
-    if (!!disruptionEndDate && !disruption.template) {
-        return isClosingOrClosed(disruptionEndDate, today);
-    }
-
-    return Progress.open;
-};
-
-export const isClosingOrClosed = (endDate: Dayjs, today: Dayjs): Progress => {
-    if (endDate.isBefore(today)) {
-        return Progress.closed;
-    } else if (endDate.diff(today, "hour") < 24) {
-        return Progress.closing;
-    }
-
-    return Progress.open;
-};
-
-export const getWorstSeverity = (severitys: Severity[]): Severity => {
-    const severityScoringMap: { [key in Severity]: number } = {
-        unknown: 0,
-        verySlight: 1,
-        slight: 2,
-        normal: 3,
-        severe: 4,
-        verySevere: 5,
-    };
-
-    let worstSeverity: Severity = Severity.unknown;
-
-    severitys.forEach((severity) => {
-        if (!worstSeverity) {
-            worstSeverity = severity;
-        } else if (severityScoringMap[worstSeverity] < severityScoringMap[severity]) {
-            worstSeverity = severity;
-        }
-    });
-
-    return worstSeverity;
-};
+export const getDashboardDisruptions = async (orgId: string, isTemplate: boolean) => {
```

Comment:
Maybe we could change the name of this function as we don't just use it on the dashboard page? 

I would have used `getAllDisruptions` but that's already the parent function's name! Maybe something like `fetchDisruptionsData`?